docs/operators/fixedWidthParserOperator.md

```diff
@@ -0,0 +1,205 @@
+User need to set the schema which describes fixed width data.
+
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *jsonSchema* | [Schema](https://github.com/apache/apex-malhar/blob/master/contrib/src/main/java/com/datatorrent/contrib/parser/FixedWidthSchema.java)  describing fixed width data. Based on schema definition the operator will parse the incoming record to object map and POJO. Valid records will be emitted as POJO / map while invalid ones are emitted on error port with error message. | String | YES | N/A |
+
+User can set this property in properties.xml in the following way
+
+```xml
+<property>
+    <name>dt.application.{ApplicationName}.operator.{OperatorName}.prop.jsonSchema</name>
+    <value>{
+      "padding": "_",
+      "alignment": "left",
+      "fields": [
+      {
+      "name": "adId",
+      "type": "Integer",
+      "length": "3",
+      "padding": "0"
+      },
+      {
+      "name": "campaignId",
+      "type": "Integer",
+      "length": "3",
+      "padding": " "
+
+      },
+      {
+      "name": "adName",
+      "type": "String",
+      "length": "10",
+      "alignment":"right"
+      },
+      {
+      "name": "bidPrice",
+      "type": "Double",
+      "length": "3"
+      },
+      {
+      "name": "startDate",
+      "type": "Date",
+      "format": "yyyy-MM-dd HH:mm:ss",
+      "length": "19"
+
+      },
+      {
+      "name": "endDate",
+      "type": "Date",
+      "format": "dd/MM/yyyy",
+      "length": "10"
+      },
+      {
+      "name": "securityCode",
+      "type": "Long",
+      "length": "5"
+      },
+      {
+      "name": "active",
+      "type": "Boolean",
+      "length": "4",
+      "trueValue": "true",
+      "falseValue": "false"
+      },
+      {
+      "name": "optimized",
+      "type": "Boolean",
+      "length": "1",
+      "trueValue": "y",
+      "falseValue": "n"
+      },
+      {
+      "name": "parentCampaign",
+      "type": "String",
+      "length": "10"
+      },
+      {
+      "name": "weatherTargeted",
+      "type": "Character",
+      "length": "1"
+      },
+      {
+      "name": "valid",
+      "type": "String",
+      "length": "10",
+      "padding":" "
+      }
+      ]
+      }
+    </value>
+</property>
+```
+Where {OperatorName} is the name of the Operator and {ApplicationName} is the name of the application.
+
+## Platform Attributes that influences operator behavior
+
+| **Attribute** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *out.TUPLE_CLASS* | TUPLE_CLASS attribute on output port which tells operator the class of POJO which need to be emitted | Class| Yes |
+
+User can set this property in properties.xml in the following way
+
+```xml
+<property>
+    <name>dt.application.{ApplicationName}.operator.{OperatorName}.port.out.attr.TUPLE_CLASS</name>
+    <value>fixedwidthparser.Ad</value>
```

Comment:
Suggest changing this to something like: com.example.fixedwidthparser.Ad
to emphasize that it should be the fully qualified class name (note that this is what is done in the example).